.github/workflows/tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [2.7, 3.6, 3.7, 3.8, 3.9, '3.10-dev', pypy2, pypy3]
+        python-version: [2.7,  3.4, 3.5, 3.6, 3.7, 3.8, 3.9, '3.10-dev', '3.11-dev', pypy2, pypy3]
```

Comment:
Python 3.4 on ubuntu-latest (20.04) is not available but available on ubuntu-18.04.

See:
* jun66j5/genshi@9f462c3
* https://github.com/jun66j5/genshi/actions/runs/1254667357
* https://github.com/actions/python-versions/blob/main/versions-manifest.json

Reply:
Assuming we want to continue supporting Python 3.4/3.5 jun66j5/genshi@9f462c3 is a good approach. jun66j5/genshi@5908c14d should be replaced by `pip install -e .` or `python setup.py develop` though. Would you mind submitting a PR?